jdk_21_maven/cs/web/triangle-ssr/src/main/resources/templates/index.html

```diff
@@ -9,13 +9,13 @@ <h2>Triangle classification</h2>
 <h2>Enter side values:</h2>
 <form action="/process" method="post">
     <label for="a">Side A:</label>
-    <input type="number" id="a" name="a" required><br><br>
+    <input type="text" id="a" name="a" required><br><br>
 
     <label for="b">Side B:</label>
-    <input type="number" id="b" name="b" required><br><br>
+    <input type="text" id="b" name="b" required><br><br>
 
     <label for="c">Side C:</label>
-    <input type="number" id="c" name="c" required><br><br>
+    <input type="text" id="c" name="c" required><br><br>
```

Comment:
regarding 
```
   @PostMapping("/process")
    public String processForm(
            @RequestParam("a") int a,
            @RequestParam("b") int b,
            @RequestParam("c") int c)
    {
```
should rather pass strings, and then use something like `Integer.parseInt()` to convert them